agent/config/beerocks_agent.conf.in

```diff
@@ -29,20 +29,17 @@ backhaul_wire_iface=@BEEROCKS_BH_WIRE_IFACE@
 [agent0]
 radio_identifier=00:00:00:00:00:00
 hostap_iface_type=WIFI_INTEL
-hostap_iface=@BEEROCKS_WLAN1_IFACE@
 hostap_ant_gain=0
 enable_repeater_mode=@BEEROCKS_REPEATER_MODE@
```

Comment:
In the task description, it is mentioned that all information is planned to be moved to the UCI.

I've decided to split it to PRs since this PR is a prerequisite for UGW prplmesh operational.

after we complete the UGW fix, we can continue the porting of the rest of the params.

Reply:
We can move everything to UCI but in appropriate hierarchies (not flat).
But I agree with Adam, not part of this PR.